jena-arq/src/main/java/org/apache/jena/sparql/algebra/optimize/TransformPathFlattern.java

```diff
@@ -32,7 +37,7 @@
  * For example,
  * <ul>
  * <li>Path seq {@literal ->} BGPs or a (sequence)
- * <li>"|" is not expanded into a union.
+ * <li>"|" is expanded into a union.
```

Comment:
Need to add new optimiser test cases that verify the modified transform is having the desired effects, and to verify it doesn't make semantically unsafe transforms as noted in other comments